### PR TITLE
Docs (Cloud Schema Modes): Add discussion of changing modes by direct editing of the schema, note modes are only configurable for dedicated instances

### PR DESCRIPTION
**Pull request by aaroncarey**
This PR details which nodes are available for dedicated instances, and describes how to edit the schema directly. Also links were added to relevant information throughout, edits were applied, and headings were changed per style guidance

Fixes DOC-328

 
 
 
 
 
 

Docs Preview: [Dgraph Preview](https://)

### CODE REVIEW

File: content/admin/schema-modes.md [hunk 1]
```diff
@@ -5,48 +5,69 @@ weight = 6
...
-[settings page](https://cloud.dgraph.io/_/settings), click the <kbd>Advanced</kbd> tab,
-and then select a mode from the **Schema Mode** list box.
+To change your schema mode on a dedicated instance, go to the [settings page](https://cloud.dgraph.io/_/settings),
+click the <kbd>General</kbd> tab, and then select a mode from the
+**Schema Mode** list box.
+
+### Changing your Schema Mode with the `/alter` endpoint
+
+{{% notice "tip" %}}
+Dgraph Labs recommends using the Dgraph Cloud [settings page](https://cloud.dgraph.io/_/settings)
```
Comment (OmarAyo) [inline content/admin/schema-modes.md:63]: I would mention that this is only possible for Dedicated

File: content/admin/schema-modes.md [hunk 2]
```diff
@@ -5,48 +5,69 @@ weight = 6
...
+to alter your schema mode for most scenarios, instead of using the `/alter` endpoint.
+{{% /notice %}}
+
+You can alter the schema mode for your dedicated instance directly in the schema
```
Comment (OmarAyo) [inline content/admin/schema-modes.md:67]: I'm not aware of this solution TBH - @tstreamDOTh @mbj36 can you please validate

File: content/admin/schema-modes.md (hunk 1 above)
Comment (aaroncarey) [reply content/admin/schema-modes.md:63]: Done!

File: content/admin/schema-modes.md (hunk 2 above)
Comment (aaroncarey) [reply content/admin/schema-modes.md:67]: My mistake - the reference should be to the /admin endpoint. Updated accordingly in the latest commit.

**Review by OmarAyo — APPROVED** [review]
LGTM

**Pull request merged**